js/src/data/test/reducer.test.js

```diff
@@ -405,6 +405,9 @@ describe( 'reducer', () => {
 		it( 'should return with policy check info', () => {
 			const data = {
 				allowed_countries: true,
+				robots_restriction: false,
```

Comment:
Based on [this comment](https://github.com/woocommerce/google-listings-and-ads/pull/1611#discussion_r943238539), [this test case](https://github.com/songchen2022/google-listings-and-ads/blob/e9e5ec823168b40908e61cea8fb2ef2e80ddba31/js/src/data/test/reducer.test.js#L404-L451) could be removed since it's already covered [here](https://github.com/songchen2022/google-listings-and-ads/blob/e9e5ec823168b40908e61cea8fb2ef2e80ddba31/js/src/data/test/reducer.test.js#L842).

Reply:
Fixed. Thanks for reminder!